src/components/pipelines-overview/list-pages/PipelineRunsForRepositoriesRow.tsx

```diff
@@ -25,7 +26,9 @@ const PipelineRunsForRepositoriesRow: React.FC<RowProps<SummaryProps>> = ({
         <ResourceLink kind="Namespace" name={obj?.group_value.split('/')[0]} />
       </td>}
       <td className={tableColumnClasses[2]}>
-        {obj.total}
+        <Link to={`/k8s/ns/${obj?.group_value.split('/')[0]}/${repositoryReference}/${obj?.group_value.split('/')[1]}/Runs`}>
```

Comment:
We call `obj?.group_value.split` now five times in this function. Can you please extract it? :smirk: 

I don't expect that obj can be undefined here, so this might work?

```ts
const [namespace, name] = obj.group_value.split('/')
```